kuiper-game/Domain/Account.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Kuiper.Domain
+{
+    public class Account
+    {
+        private decimal _balance;
+        private IList<Transaction> _transactions;
+
+        [JsonConstructor]
+        public Account(decimal balance)
+        {
+            _balance = balance;
+            _transactions = new List<Transaction>();
+        }
+
+        public Account(decimal balance, IList<Transaction> transactions)
+        {
+            _balance = balance;
+            _transactions = transactions;
+        }
+
+        public decimal Balance { get { return _balance; } }
+
+        public decimal Deposit(decimal amount) {
+            _balance += amount;
+            _transactions.Add(new Transaction(DateTime.Now, TransactionType.Deposit, amount));
```

Comment:
unless it's intentionally RealTime, use GameTime.Now()

Reply:
Now running on GameTime ™️